CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.6.0 - unreleased
+
+### Fixed
+* [#752](https://github.com/nextcloud/deck/issues/752) Print Style Error
```

Comment:
Could you remove the changelog change? The file will be updated during the release process automatically from the changes that then got merged for the release :)

Reply:
Ah - didn't know that. Done